packages/manager/apps/public-cloud/src/index.html

```diff
@@ -25,6 +25,9 @@
                     </div>
                 </div>
             </div>
+            <a data-ng-if="$ctrl.feedbackUrl" href="{{ $ctrl.feedbackUrl }}" target="blank" style="position: fixed; right: 2rem; bottom: 2rem;">
```

Comment:
Consider adding `rel="noopener"` would be great.

```suggestion
            <a data-ng-if="$ctrl.feedbackUrl" href="{{ $ctrl.feedbackUrl }}" target="_blank" style="position: fixed; right: 2rem; bottom: 2rem;">
```